util/topgen/templates/chiplevel.sv.tpl

```diff
@@ -630,13 +630,10 @@ module chip_${top["name"]}_${target["name"]} (
   logic sck_monitor;
 
   // observe interface
-  logic [8-1:0] fla_obs;
-  logic [8-1:0] otp_obs;
+  logic [7:0] fla_obs;
```

Comment:
nit: is this supposed to be `flash_obs`?

Reply:
no